opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -1318,14 +1318,17 @@ const mergeEntitiesRaw = async (context, user, targetEntity, sourceEntities, tar
 
 const loadMergeEntitiesDependencies = async (context, user, entityIds) => {
   const data = { [INTERNAL_FROM_FIELD]: [], [INTERNAL_TO_FIELD]: [] };
+  const rawDependencies = [];
   for (let entityIndex = 0; entityIndex < entityIds.length; entityIndex += 1) {
     const entityId = entityIds[entityIndex];
     // Internal From
     const listFromCallback = async (elements) => {
-      const findArgs = { toMap: true, baseData: true };
+      const findArgs = { toMap: true };
```

Comment:
why remove baseData param ? (same question for the other places where baseData has been removed.

Reply:
unless I missed something during my tests, baseData didn't include markings, which are needed for the userFilterStoreElements in mergeEntities